terraform/modules/openstack_host/inputs.tf

```
@@ -0,0 +1,66 @@
+variable "project_name" {
+  description = "A prefix for names of objects created by this module"
+  default     = "st"
+}
+
+variable "availability_zone" {
+  description = "Availability zone where the instance is created"
+  type        = string
+  default     = "nova"
+}
+
+variable "name" {
+  description = "Symbolic name of this instance"
+  type        = string
+}
+
+variable "image" {
+  description = "Openstack Image/Glance ID"
+  default     = "af0caca5-fd24-4340-a84a-e60e088bc92d" # Centos 7 on OVH GRA7 Region
+}
+
+variable "flavor" {
+  description = "Openstack Flavor name"
+  default     = "b2-7"
+}
+
+variable "op_keypair" {
```

I'd prefer not to prefix all backend-specific variable names, I think we can just use "keypair" for simplicity here:

```suggestion
variable "keypair" {
```